src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -128,18 +128,17 @@ public void LogDependency_SinksToApplicationInsights_ResultsInDependencyTelemetr
             {
                 ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
 
-                string dependencyType = _bogusGenerator.Name.FullName();
+                string dependencyType = _bogusGenerator.Lorem.Word();
                 object dependencyData = _bogusGenerator.Finance.Account();
                 bool isSuccessful = _bogusGenerator.PickRandom(true, false);
-                DateTimeOffset startTime = _bogusGenerator.Date.PastOffset();
                 TimeSpan duration = _bogusGenerator.Date.Timespan();
                 Dictionary<string, object> telemetryContext = CreateTestTelemetryContext();
 
                 // Act
-                logger.LogDependency(dependencyType, dependencyData, isSuccessful, startTime, duration, telemetryContext);
+                logger.LogDependency(dependencyType, dependencyData, isSuccessful, DateTimeOffset.UtcNow, duration, telemetryContext);
```

Comment:
As discussed, let's see if we have to do this because of [`StartTime`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.applicationinsights.extensibility.implementation.operationtelemetry.starttime?view=azure-dotnet#Microsoft_ApplicationInsights_Extensibility_Implementation_OperationTelemetry_StartTime) and if [`Timestamp`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.applicationinsights.datacontracts.dependencytelemetry.timestamp?view=azure-dotnet#Microsoft_ApplicationInsights_DataContracts_DependencyTelemetry_Timestamp) works better

Reply:
The `DateTimeOffset` is passed allong the ctor of the `DependencyTelemetry` with the `startTime` argument and is passed along to `Timestamp`.
![image](https://user-images.githubusercontent.com/9039753/78884476-47985680-7a5b-11ea-8891-6e339add1b5a.png)

Reply:
This really is odd. How far in the past was the initial test?

Reply:
Past previous year even.

Reply:
I changed all dependency tracking with a recent start time.

Reply:
Perfect, thanks!

Does the tracking still show up like it should in Application Insights?

Reply:
Yes, not it is ;), changed it to the current day.

Reply:
This is odd, you have to be a wizard. This makes me wonder, should we guard for this stuff to help our consumers? What do you think?

Reply:
We coculd try to guard it, but it is only for Application Insights sink + we don't know exactly where the boundaries are. I didn't find any guards on the "client"-side Application Insights sink, so I guess it's something that is only present in the "server"-side Application Insights.

Reply:
You are correct! Resolving.